product_docs/docs/edbcloud/beta/administering_cluster/01_portal_access.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: "Managing Portal Access"
+---
+
+EDB Cloud uses Azure Active Directory to authenticate users and role based access controls to grant users access to different parts of the application.
+
+## Organizations
+
+Each subscribed customer has a unique *organization*. Each organization’s database clusters are deployed and managed in the customer's cloud account. This ensures complete segregation of customer data between organizations.
+
+Each EDB Cloud organization is associated with an Azure Active Directory (AD) tenant. Azure AD establishes the identity of users that belong to an organization. After Azure AD is linked during subscription, users that belong to that AD can login with minimum access privileges.
+
+EDB Cloud supports role based access control policies. A user with the **owner** role may assign roles to other users in the same organization.
```

Comment:
The mix of italics, bold, and preformatted in this page is still a little jarring to me - e.g., "owner" is bolded here, but italicized down on L19.

Reply:
@ba-edb, I'm trying out not using font treatments for roles and permissions.